webapp/src/components/Controls/Controls.tsx

```diff
@@ -101,7 +106,11 @@ const Controls: React.FC<Props> = ({
 
   const handleClearFilters = () => {
     history.push(
-      `${baseUrl}${constructSearchString({ ...pagination, ...pipeline })}`
+      `${baseUrl}${constructSearchString({
+        ...pagination,
+        ...pipeline,
+        withoutPostprocessing: undefined,
```

Comment:
This line doesn't do anything. Anyway, what I proposed was
```suggestion
        withoutPostprocessing,
```

Reply:
It leaves me an error if I just give it like this     history.push(
      `${baseUrl}${constructSearchString({
        ...pagination,
        ...pipeline,
        withoutPostprocessing,
      })}`
    );

Reply:
Sorryyy! I was confused between the object `withoutPostprocessing` and the boolean `withoutPostprocessing`. Another evidence that we should rename the object! 😉 
I meant:
```suggestion
        ...withoutPostprocessing,
```

Reply:
ye true!.. working on that already

Reply:
I tried options to just add "withoutPostprocessing" to the above line of code but It stops me with an error "TS18004: No value exists in scope for the shorthand property 'withoutPostprocessing'. Either declare one or provide an initializer." any help/suggestion on this would be very much helpful

Reply:
Now that you renamed `withoutPostprocessing` to `postprocessing`, it should be
```suggestion
        ...postprocessing,
```

Reply:
still clear filters doest reset with it

Reply:
I tried and it works as expected.

Reply:
added the necessary changes